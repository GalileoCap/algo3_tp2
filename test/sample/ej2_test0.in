3
8 7
1 2 2
2 4 7
4 3 10
3 5 6
5 6 10
6 7 8
7 8 2
8 7
1 2 2
2 4 7
4 3 10
3 5 6
5 6 10
6 7 8
7 8 1
6 7
1 2 6
2 3 10
3 4 9
1 4 5 
4 5 4
4 6 8
6 5 2
