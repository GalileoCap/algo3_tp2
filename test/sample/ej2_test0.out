24
31
20
